cmake/FindCMSIS.cmake

```diff
@@ -156,8 +156,20 @@ IF(STM32_CHIP_TYPE)
         CMAKE_FIND_ROOT_PATH_BOTH
     )
     LIST(APPEND CMSIS_SOURCES ${CMSIS_STARTUP_SOURCE_FILE})
+
+    SET(CMSIS_COMPILE_DEFINITIONS "STM32F${STM32_CHIP_TYPE}")
```

Comment:
Is it normal to have a F here? STM32**F**${STM32_CHIP_TYPE}

Reply:
It's an error. I just recently caught this bug when in one of the projects I started using L0 family devices.